app/src/components/OrderBookSnapshot.js

```diff
@@ -31,12 +45,17 @@ class OrderBookSnapshot extends Component {
      * @desc Handles the date change for the TextField date picker
      * @param event The event object that caused the call
      */
-    handleChangeDate = (event) => {
+    handleChangeDate = event => {
         let selectedDateNano = parseInt(dateStringToEpoch(event.target.value + ' 00:00:00'));
 
-        this.setState({
-            selectedDateNano,
-        }, () => { this.handleChangeDateTime(); });
+        this.setState(
+            {
+                selectedDateNano
+            },
+            () => {
+                this.handleChangeDateTime();
+            }
```

Comment:
It's nice to have a linter on save but I don't know if the styling (in cases like above) is necessary. what do you think?

Reply:
it's always better to have clear linting rules so that style remains consistent. Without defining it, it could lead to some editors formatting the same piece of code in different ways. The problem of large diffs can be solved with a more complete .eslintrc file

Reply:
Perhaps if we use a preset, we won't have any more issues with larger diffs since there will be no ambiguity in terms of formatting.

Reply:
Gonna look at this on my newest pr